### PR TITLE
Add data file

### PR DESCRIPTION
**Pull request by drwpow**
Adds starter data for summit peaks, in JSON format, with the following structure:

- `name`: human-readable name
- `slug`: Unique ID for each peak (that’s also human readable-ish)
- `elevation`: Elevation, in feet. Left as a JavaScript `Number` so we can perform Maths™ on it
- `lat`: latitude, expressed as a `Float`, also for the purposes of doing Maths™
- `lng`: longitude

I was able to retrieve these easily from Google Maps by right-clicking on a peak and selecting “What’s here?”—the lat/lng shows up. The elevations I grabbed from Wikipedia.

This is just the start; we’ll add several more before we place them on a map.

### CODE REVIEW

**Comment by dangodev** [comment]
So from here @jmjcobs, you can review this PR, and respond with any feedback/questions/changes.

Then from here, I’ll pass the data sourcing off to you—which peaks do we want to have? Probably would be good to get the major ones in the Teton range. Then we can do the fun part and add them on Google Maps.

JSON has become a really good data standard—it’s readable by any programming language, and it’s pretty minimal syntax surrounding data. It’s always a good starting point—whether you‘re working in JS, Go, or any language—until you have a reason to port something into a database.

YAML is another good option, and is very similar, but that’s hard to parse for JS. And friends don’t let friends ever use XML for data—that’s just asking for trouble.

**Review by jmjcobs — APPROVED** [review]
@dangodev this data really peaks my interest!  

Looks good to me!  Go a head and merge away.
